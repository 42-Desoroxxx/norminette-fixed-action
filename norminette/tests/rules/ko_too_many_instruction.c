int a; int b;

void f(void)
{
	write(1, "a", 1); write(1, "a", 1);
}
