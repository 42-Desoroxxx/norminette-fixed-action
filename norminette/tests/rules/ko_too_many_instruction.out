tests/rules/ko_too_many_instruction.c: KO!
	TOO_MANY_INSTR (line: 1, col: 10):	Too many instructions on a single line
	MISALIGNED_VAR_DECL (line: 1, col: 20):	Misaligned variable declaration
	TOO_MANY_INSTR (line: 5, col: 23):	Too many instructions on a single line
	TOO_FEW_TAB (line: 5, col: 23):	Missing tabs for indent level
	TOO_FEW_TAB (line: 5, col: 23):	Missing tabs for indent level
